cmd/status-cli/util.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/status-im/status-go/api"
+	"github.com/status-im/status-go/logutils"
+	"github.com/status-im/status-go/params"
+	"github.com/status-im/status-go/protocol/requests"
+	"github.com/status-im/status-go/services/wakuv2ext"
+
+	"github.com/urfave/cli/v2"
+	"go.uber.org/zap"
+)
+
+func setupLogger(file string) *zap.Logger {
+	logFile := fmt.Sprintf("%s.log", strings.ToLower(file))
+	logSettings := logutils.LogSettings{
+		Enabled:         true,
+		MobileSystem:    false,
+		Level:           "DEBUG",
+		File:            logFile,
+		MaxSize:         100,
+		MaxBackups:      3,
+		CompressRotated: true,
+	}
+	if err := logutils.OverrideRootLogWithConfig(logSettings, false); err != nil {
+		logger.Fatalf("Error initializing logger: %v", err)
+	}
+
+	newLogger := logutils.ZapLogger()
+
+	return newLogger
+}
+
+func withHTTP(port int) params.Option {
+	return func(c *params.NodeConfig) error {
+		c.APIModules = "waku,wakuext,wakuv2,permissions,eth"
+		c.HTTPEnabled = true
+		c.HTTPHost = "127.0.0.1"
+		c.HTTPPort = port
+
+		return nil
+	}
+}
+
+func startMessenger(cCtx *cli.Context, name string, port int) (*StatusCLI, error) {
+	namedLogger := logger.Named(name)
+	namedLogger.Info("starting messager")
+
+	_ = setupLogger(name)
+
+	path := fmt.Sprintf("./test-%s", strings.ToLower(name))
+	err := os.MkdirAll(path, os.ModePerm)
```

Comment:
Since accounts seem to be created each time the cmd is executed, consider using a folder in /tmp with os.MkdirTemp

Reply:
I agree with Richard.
And if the path is hard to find, just print the path to console.
Or use the `Application Support` directory.

Also we will not need the custom `.gitignore` lines for logs then.